docs/gaia/delegator-guide-cli.md

```diff
@@ -0,0 +1,406 @@
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+### Creating an account
+
+To create an account, you just need to have `gaiacli` installed. Before creating it, you need to know where you intend to store and interract with your private keys. The best options are to store them in an offline dedicated computer or a ledger device. Storing them on your regular online computer involves more risk.
+
+#### Using a ledger device
+
+When you initialize your ledger, a 24-words mnemonic is generated and stored in the device. This mnemonic is compatible with Cosmos and Cosmos Accounts can be derived from it. Therefore, all you have to do is make your ledger compatible with `gaiacli`. To do so, you need to go through the following steps:
+
+1. Download the Ledger Live app [here](https://www.ledger.com/pages/ledger-live)
+2. Connect your ledger via USB and update to the latest firmware
+3. Go to the ledger live app store, and download the "Cosmos" application (this can take a while)
+4. Navigate to the Cosmos app on your ledger device
+
+Then, to create an account, use the following command:
+
+```bash
+gaiacli keys add <yourAccountName> --ledger 
+```
+
+- `<yourKeyName>` is the name of the account. It is a reference to the account number used to derive the key pair from the mnemonic. You will use this name to identify your account when you want to send a transaction.
+- You can add the optional `--account` flag to specify the path (`0`, `1`, `2`, ...) you want to use to generate your account. By default, account `0` is generated. 
+
+#### Using a computer 
+
+::: warning
+**NOTE: It is more secure to perform this action on an offline computer**
+:::
+
+To generate an account, just use the following command:
+
+```bash
+gaiacli keys add <yourKeyName>
+```
+
+The command will generate a 24-words mnemonic and account `0` at the same time. You will be prompted to input a passphrase that is used to encrypt the private key of account `0` on disk. Each time you want to send a transaction, this password will be required. If you loose the password, you can always recover the private key with the mnemonic. 
+
+::: danger
+**NEVER LOOSE YOUR MNEMONIC. WRITE IT DOWN ON A PIECE OF PAPER AND STORE IT SOMEWHERE SAFE. IF YOU LOOSE IT, THERE IS NO WAY TO RETRIEVE IT. IF SOMEONE GAINS ACCESS TO IT, THEY GAIN ACCESS TO ALL THE ASSOCIATED ACCOUNTS**
+::: 
+
+::: warning
+After you have secured your mnemonic (triple check!), you can delete bash history to ensure no one can retrieve it:
```

Comment:
Does bash store input by default? We should change the CLI so that this isn't the case - as with e.g. password input.

Reply:
well the mnemonic is displayed on bash when you create an account, so I assume it gets stored?